lib/api_hammer/sinatra/halt.rb

```diff
@@ -5,7 +5,7 @@ module Sinatra
     module Halt
       # halt and render the given body 
       def halt(status, body, render_options = {})
-        throw :halt, format_response(status, body, headers)
+        throw :halt, format_response(status, body)
```

Comment:
#headers is a method on sinatra base, this would have retrieved the already-set headers and re-set those to what they already were. so no actual effect, but this was an error.